lisa/node.py

```diff
@@ -1215,7 +1215,8 @@ def create_nics(node: Node) -> Nics:
     """
     Returns a Nics object for the node based on the OS type.
     """
-    if isinstance(node.os, BSD):
+    os = node.execute(cmd="uname", no_error_log=True).stdout
```

Comment:
Add comments above to explain why we don't use `isinstance`. So in future, it won't be changed back.